redirects-list.json

```diff
@@ -38,6 +38,9 @@
   "^/features$                                                                            /doc/use-cases 302",
 
   "^/doc/install/plugins(/.*)?$                                                           /doc/install/ide-plugins$1",
+  "^/doc/install/completion#bash-completion-on-macos$                                     /doc/install/completion?tab=Bash-macOS",
+  "^/doc/install/completion#bash-completion-on-debianubuntu$                              /doc/install/completion?tab=Bash-Debian-Ubuntu",
+  "^/doc/install/completion#zsh-completion$                                               /doc/install/completion?tab=Zsh",
```

Comment:
I don't think that the URL anchors make it to the router. Can you confirm @iterative/websites ? Thanks

Reply:
In any case we don't have any links to these sections directly from our docs so I wouldn't bother with these.

Reply:
```suggestion
  "^/doc/install/plugins(/.*)?$                                                           /doc/install/ide-plugins$1",
```